app/models/episode_import.rb

```diff
@@ -86,7 +86,7 @@ def set_audio_metadata!
   end
 
   def set_file_resources!
-    episode.contents = audio_content_params
+    episode.media = audio_content_params
```

Comment:
`audio` may be a bit of a misnomer here?  RSS can contain enclosures for other media as well.

Reading down in `def entry_audio_files` ... it looks like it either pulls media urls from the `config[:audio]` (which IIRC is just there in case we want to manually override something).  Or more often, it's the `enclosure_url` - which could be a video url.